server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java

```diff
@@ -686,9 +691,32 @@ public List<String> getAsyncDownloadUrl(Long id, List<String> objectIds) {
         return downloadUrls;
     }
 
-    public String getLog(Long scheduleId, Long taskId, OdcTaskLogLevel logLevel) {
-        nullSafeGetByIdWithCheckPermission(scheduleId);
-        return scheduleTaskService.getLogWithoutPermission(taskId, logLevel);
+    public String getFullLogDownloadUrl(Long scheduleId, Long scheduleTaskId, boolean skipAuth) {
```

Comment:
add skipAuth parameter is a bad practice, use separated method for skipAuth instead

if add parameter, then the authorization mechanism will not able to check permission risk

Reply:
done